### PR TITLE
Add Ticket.search() and Ticket.update() methods.

### PR DESCRIPTION
**Pull request by ekoval**
Add Ticket.search() and Ticket.update() methods.

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: teamsupport/models.py [hunk 1]
```diff
@@ -64,6 +64,15 @@ def create(cls, user_email, user_first_name, user_last_name, title,
         return ticket
 
     @classmethod
+    def search(cls, **query_params):
+        client = cls.get_client()
+        return [Ticket(data=data)
+                for data in client.search_tickets(**query_params)]
+
+    def update(self, **data):
+        self.data = self.client.update_ticket(self.id, data)
```
Comment (RayeN) [inline teamsupport/models.py:73]: I would return `self` here, just for convenience.

File: integration_tests/test_tickets.py [hunk 2]
```diff
@@ -78,3 +87,60 @@ def test_new_contact_is_not_created(self):
     def tearDown(self):
         self.ticket.delete()
         self.contact.delete()
+
+
+class TestSearchTickets(TestTicket):
+    def setUp(self):
+        self.created_ticket_name = self.random_name
```
Comment (RayeN) [inline integration_tests/test_tickets.py:90]: can we just use `uuid` for that :)

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: teamsupport/models.py (hunk 1 above)
Comment (RayeN) [inline teamsupport/models.py:73]: thanks.

**Comment by RayeN** [comment]
:+1: Looks good to me.  Comment regarding tests is optional.

**Pull request merged**